test/end-to-end/pages/deceased/details.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const commonContent = require('app/resources/en/translation/common');
+const pageUnderTest = require('app/steps/ui/deceased/details/index');
+
+module.exports = function (firstName, lastName, dob_day, dob_month, dob_year, dod_day, dod_month, dod_year) {
+    const I = this;
+    I.seeCurrentUrlEquals(pageUnderTest.getUrl());
+
+    I.fillField('#firstName', firstName);
+    I.fillField('lastName', lastName);
```

Comment:
Shouldn't we be using the id for the `lastName` field as we are with the other fields?